eZ/Publish/API/Repository/Values/Content/Location.php

```diff
@@ -56,6 +58,35 @@
     const STATUS_PUBLISHED = 1;
 
     /**
+     * Map for Location sort fields to their respective SortClauses.
+     *
+     * Those not here (class name/identifier and modified subnode) are
+     * missing/deprecated and will most likely be removed in the future.
+     */
+    const SORT_FIELD_MAP = [
```

Comment:
We're making this a part of API by putting it here, and I see no reason to do that. I would rather put this in the `Core` implementation, together with `SORT_ORDER_MAP`. Method `getSortClauses()` should then be implemented there as well. Maybe also mark it there as `internal` until we have PHP 7.1 with constant visibility as minimum :)